src/app/tests/suites/TestAccessControlCluster.yaml

```diff
@@ -0,0 +1,167 @@
+                      Targets:
+                          [
+                              { Cluster: null, Endpoint: 1, DeviceType: null },
+                              { Cluster: 2, Endpoint: null, DeviceType: null },
+                              { Cluster: 3, Endpoint: 4, DeviceType: null },
+                          ],
+                  },
+                  {
+                      FabricIndex: 1,
+                      Privilege: 3, # operate
+                      AuthMode: 3, # group
+                      Subjects: [0x0101, 0x0102, 0xAAAA, 0xBBBB],
+                      Targets:
+                          [
+                              { Cluster: null, Endpoint: 2, DeviceType: null },
+                              { Cluster: 3, Endpoint: null, DeviceType: null },
+                              { Cluster: 4, Endpoint: 5, DeviceType: null },
+                          ],
+                  },
+              ]
+
+    - label: "Read"
+      command: "readAttribute"
+      attribute: "ACL"
+      response:
+          value: [
+                  {
+                      FabricIndex: 1,
+                      Privilege: 5, # administer
+                      AuthMode: 2, # case
+                      Subjects: [nodeId, 1234, 5678, 9090],
+                      Targets:
+                          [
+                              { Cluster: null, Endpoint: 0, DeviceType: null },
+                              { Cluster: 1, Endpoint: null, DeviceType: null },
+                              { Cluster: 2, Endpoint: 3, DeviceType: null },
+                          ],
+                  },
+                  {
+                      FabricIndex: 1,
+                      Privilege: 1, # view
+                      AuthMode: 2, # case
+                      Subjects: [1234, 5678, 0x8080, 0xFFFF],
+                      Targets:
+                          [
+                              { Cluster: null, Endpoint: 1, DeviceType: null },
+                              { Cluster: 2, Endpoint: null, DeviceType: null },
+                              { Cluster: 3, Endpoint: 4, DeviceType: null },
+                          ],
+                  },
+                  {
+                      FabricIndex: 1,
+                      Privilege: 3, # operate
+                      AuthMode: 3, # group
+                      Subjects: [0x0101, 0x0102, 0xAAAA, 0xBBBB],
+                      Targets:
+                          [
+                              { Cluster: null, Endpoint: 2, DeviceType: null },
+                              { Cluster: 3, Endpoint: null, DeviceType: null },
+                              { Cluster: 4, Endpoint: 5, DeviceType: null },
+                          ],
+                  },
+              ]
+
+    - label: "Write"
+      command: "writeAttribute"
+      attribute: "ACL"
+      arguments:
+          value: [
+                  {
+                      FabricIndex: 1,
```

Comment:
Please use `FabricIndex: 0` on the writes (since it gets ignored on the server anyway).

Reply:
It's easier to copy/paste the value from write to read test steps if they are the same. But it's also easier to just do the recommendation for now, I'll be back in this file soon and will be restructuring anyways.